examples/extend_modern_web_ui_cache_stats.py

```diff
@@ -0,0 +1,209 @@
+"""
+This is an example of a locustfile that uses Locust's built in event and web
+UI extension hooks to track the sum of Varnish cache hit/miss headers
+and display them in the web UI.
+"""
+
+from locust import HttpUser, TaskSet, between, events, task, web
+
+import json
+import os
+from html import escape
+from time import time
+
+from flask import Blueprint, jsonify, make_response, render_template, request
+
+
+class MyTaskSet(TaskSet):
+    @task(1)
+    def miss(l):
+        """MISS X-Cache header"""
+        l.client.get("/response-headers?X-Cache=MISS")
+
+    @task(2)
+    def hit(l):
+        """HIT X-Cache header"""
+        l.client.get("/response-headers?X-Cache=HIT")
+
+    @task(1)
+    def noinfo(l):
+        """No X-Cache header (noinfo counter)"""
+        l.client.get("/")
+
+
+class WebsiteUser(HttpUser):
+    host = "http://httpbin.org"
+    wait_time = between(2, 5)
+    tasks = [MyTaskSet]
+
+
+# @see https://docs.acquia.com/acquia-cloud-platform/performance/varnish/headers
```

Comment:
Maybe we can simply explain this in a comment or point to a blog post, rather than referencing a companies documentation?

Reply:
True, I'll add a short comment, but I don't think that details about Varnish headers should be explain here.
I should also fix the link, here the correct one https://docs.varnish-software.com/tutorials/hit-miss-logging/